### DISCUSSION TITLE
Missing database in menu

### DISCUSSION THREAD

**Discussion by andriibezkorovainyi** [dbeaver/dbeaver#21407] [Q&A]
### Description

My added database is disappearing from the databases list, though it persists on the server(if i make requests i got proper responses).  

### DBeaver Version

Community Edition 23.2.1.202309250303

### Operating System

macOS Ventura 13.0

### Database and driver

1. Databse PostgreSQL 15.2
2. Driver PostgreSQL JDBC Driver [42.5.2]

### Steps to reproduce

1. Created database named db1, for example
2. Now i have default postgres and db1 databases 
3. Then i was using db1 for 1 day and closed the app
4. Next day I launch DBeaver, connect to the server(local) and db1 is gone
5. If i check the server with pgadmin - it is shown

### Additional context

No extensions

**Comment by LonwoLonwo** [comment]
Hello @andriibezkorovainyi 

Please check that this setting is enabled in your connection settings:

![2023-10-02 18_03_52-Connection _postgres12local_ configuration](https://github.com/dbeaver/dbeaver/assets/45152336/eeb95d0f-7485-4ef3-b13a-c9fb6a109031)

**Comment by andriibezkorovainyi** [comment]
Hi, @LonwoLonwo. This helped, thank you very much  🤗, but i must say, the behaviour like that isnt enough obvious, regarding to the fact, that these settings are only changable while establishing connection rather then edit existing one. So, maybe, this option should be applied by default on my opinion. But im nove with the rest of the software features :)

**Comment by LonwoLonwo** [comment]
You can change this setting during the new connection creation.

You can change it.

You can set it for all new connections in Preferences:

![2023-10-02 18_38_56-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/6a58d3c1-8db8-41e4-b127-55da9d830f41)

But unfortunately, we can enable it by default. It will cause many problems for other users, including decreasing performance.
